package.json

```diff
@@ -8,8 +8,8 @@
     "build:functions": "next lint -d functions/src && yarn --cwd functions build",
     "check-formatting": "prettier --check .",
     "check-types": "tsc --noEmit",
-    "deploy:backend:dev": "firebase --project=default deploy --only firestore,functions,storage",
-    "deploy:backend:prod": "firebase --project=prod deploy --only firestore,functions,storage",
+    "deploy:backend:dev": "firebase --project=default deploy --only firestore,functions:maple,storage",
+    "deploy:backend:prod": "firebase --project=prod deploy --only firestore,functions:maple,storage",
```

Comment:
Now that we also have Python, and it is a little janky right now, just deploy the maple functions and skip the python ones.